packages/icons/deprecated.yml

```diff
@@ -1,3 +1,5 @@
 deprecated:
 - name: kubernetes
   reason: The current kubernetes icon has been replaced by logo--kubernetes
+- name: direct-link
+  reason: The current icon has been replaced with the correct version of the App icon
```

Comment:
Should `direct-link` no longer be used? If so, what should be used in it's place?

If this is only an artwork change and the name is staying the same, I don't believe we need to have it listed as a deprecation.

Reply:
@tay1orjones correct, `direct-link` should no longer be used. There's a whole suite of "direct link" related icons in this batch but the one that would be the 1:1 substitution is: `ibm-cloud--direct-link-2--dedicated`

<img width="469" alt="Screen Shot 2022-09-06 at 2 54 31 PM" src="https://user-images.githubusercontent.com/16782944/188716371-dd705846-d1cb-47ae-9a3d-95057ba60a23.png">

Reply:
Would it be possible to make a suggestion like that in the reason comment? Something like "The current icon has been replaced with the correct version of the App icon, use the most appropriate ibm-cloud--direct-link icon variant instead"

Reply:
@abbeyhrt yup, just updated the .yml file with your suggested update